setup.py

```diff
@@ -5,14 +5,16 @@
 
 setup(
     name='keras-eval',
-    version='0.0.33',
+    version='0.0.34',
     description='An evaluation abstraction for Keras models.',
     author='Triage Technologies Inc.',
     author_email='ai@triage.com',
     url='https://www.triage.com/',
     packages=find_packages(exclude=['tests', '.cache', '.venv', '.git', 'dist']),
     install_requires=[
-        'Keras>=2.2.0',
+        'numpy==1.14.5',
```

Comment:
Why we need to fix `numpy` version? 🤔

Reply:
If not there is an incompatible version that gets installed.

Reply:
I'm just curious what's the package that is incompatible?

Reply:
I think `tensorflow 1.10` if I remember right as the default version of numpy is higher than tf can  take.